scripts/qa_crowbarsetup.sh

```diff
@@ -1962,7 +1962,7 @@ function set_crowbar_ipmi_ip_addrs
     IFS=. read ip1 ip2 ip3 ip4 <<< "${bmc_values[0]}"
     local bmc_net="$ip1.$ip2.$ip3"
     local i
-    for i in $(seq 1 $nodenumbertotal); do
+    for i in $(seq 1 $nodenumber); do
```

Comment:
The nodenumbertotal here was meant to allow physical clouds of 5 nodes, where we only use 2 and turn off the other 3 via IPMI so that IP addrs assigned in the previous run do not collide with those of the active nodes.